packages/jaeger-ui/src/components/TraceDiff/TraceDiffGraph/drawNode.css

```diff
@@ -17,16 +17,20 @@ limitations under the License.
 .DiffNode {
   background: #bbb;
   border: 1px solid #777;
-  box-shadow: 0 0px 3px rgba(0, 0, 0, 0.2);
+  box-shadow: 0 0px 3px 3px rgba(0, 0, 0, 0.2);
```

Comment:
I changed this because when looking at even a moderate diff, I couldn't see any box-shadow on the non-matches. But that could be in part be because of the darker background.

Reply:
changed to `1px` with the background change. still seems fairly subtle but it's not that crucial / can be tweaked further independently of the rest of the PR. I'm also RG colorblind so that's certainly not helping my ability to see red and green box shadows.